migration-guide/src/explorer-api.md

```diff
@@ -0,0 +1,2904 @@
+
+Get address information specific to a block.
+
+=== "cardano-rest"
+
+    ```console
+    $ https://explorer.cardano-testnet.iohkdev.io/api/block/02788224fb087d9a5ed48a12a3db741ad7bd4429a63e7e5b6bc69d2f2956d205/address/addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5
+    ```
+
+    <details>
+      <summary>see JSON response</summary>
+    ```json
+    {
+      "Right": {
+        "caAddress": "addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5",
+        "caType": "CPubKeyAddress",
+        "caChainTip": {
+          "ctBlockNo": 2225979,
+          "ctSlotNo": 16239920,
+          "ctBlockHash": "0ad0938aab4a1db979fa84fc38ca742dd3b1d6ff548e50465fbf9b7a42d0b518"
+        },
+        "caTxNum": 1,
+        "caBalance": {
+          "getCoin": "590707773"
+        },
+        "caTotalInput": {
+          "getCoin": "590707773"
+        },
+        "caTotalOutput": {
+          "getCoin": "0"
+        },
+        "caTotalFee": {
+          "getCoin": "172101"
+        },
+        "caTxList": [
+          {
+            "ctbId": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+            "ctbTimeIssued": 1610575838,
+            "ctbInputs": [
+              {
+                "ctaAddress": "addr_test1qp6dnz3gs84zn7cjx3mrhquj99zfd0acqh9qjldeec9rp4zkvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqs553m2",
+                "ctaAmount": {
+                  "getCoin": "591879874"
+                },
+                "ctaTxHash": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+                "ctaTxIndex": 1
+              }
+            ],
+            "ctbOutputs": [
+              {
+                "ctaAddress": "addr_test1qrvf30r0e6r8zjzmv22a4r3h8kzj2xf3l2ekezvqd66mcj6kvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqrfd5g5",
+                "ctaAmount": {
+                  "getCoin": "590707773"
+                },
+                "ctaTxHash": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+                "ctaTxIndex": 0
+              },
+              {
+                "ctaAddress": "addr_test1qrlh7p9th5c9ps938ry05vq96j92lzuhqr29v46caydf2wzkvlatzplcfr8afde6wsr6weskqr8k3u80e957ecmkvkhqt48df3",
+                "ctaAmount": {
+                  "getCoin": "1000000"
+                },
+                "ctaTxHash": "16e73c7f887825152831eaa656c496aa45f69c755bb6e03a3454555e0c790196",
+                "ctaTxIndex": 1
+              }
+            ],
+            "ctbInputSum": {
+              "getCoin": "591879874"
+            },
+            "ctbOutputSum": {
+              "getCoin": "591707773"
+            },
+            "ctbFees": {
+              "getCoin": "172101"
+            }
+          }
+        ]
+      }
+    }
+    ```
+    </details>
+
+=== "cardano-graphql"
+
+    !!! info
+        This query is WIP. A few fields are still missing.
```

Comment:
There are a few problem with this query, also see https://github.com/input-output-hk/cardano-graphql/pull/401/files#r556273888